wlutil/br/buildroot-config

```diff
@@ -1,2983 +1,734 @@
+# BR2_KAGE_PHP_EXT_ZIP is not set
+# BR2_KAGE_PHP_GEOIP is not set
+# BR2_KAGE_PHP_GNUPG is not set
+# BR2_KAGE_PHP_IMAGICK is not set
+# BR2_KAGE_PHP_MEMCACHED is not set
+# BR2_KAGE_PHP_SAPI_APACHE is not set
+# BR2_KAGE_PHP_SAPI_CLI is not set
+# BR2_KAGE_PHP_SAPI_FPM is not set
+# BR2_KAGE_PHP_SSH2 is not set
+# BR2_KAGE_PHP_YAML is not set
+# BR2_KAGE_PHP_ZMQ is not set
+# BR2_KAGE_PHYSFS is not set
+# BR2_KAGE_PIXZ is not set
+# BR2_KAGE_POLKIT is not set
+# BR2_KAGE_POPPLER is not set
+# BR2_KAGE_POSTGRESQL is not set
+# BR2_KAGE_POWERTOP is not set
+# BR2_KAGE_PROTOBUF is not set
+# BR2_KAGE_PROTOBUF_C is not set
+# BR2_KAGE_PSMISC is not set
+# BR2_KAGE_PSPLASH is not set
+# BR2_KAGE_PUGIXML is not set
+# BR2_KAGE_PULSEAUDIO is not set
+# BR2_KAGE_PUTTY is not set
+# BR2_KAGE_PYTHON is not set
+# BR2_KAGE_PYTHON3 is not set
+# BR2_KAGE_QHULL is not set
+# BR2_KAGE_QLIBC is not set
+# BR2_KAGE_QPDF is not set
+# BR2_KAGE_QT5 is not set
+# BR2_KAGE_QUOTA is not set
+# BR2_KAGE_RANGER is not set
+# BR2_KAGE_RAPIDJSON is not set
+# BR2_KAGE_RAUC is not set
+# BR2_KAGE_RESTORECOND is not set
+# BR2_KAGE_RIEMANN_C_CLIENT is not set
+# BR2_KAGE_RPM is not set
+# BR2_KAGE_RRDTOOL is not set
+# BR2_KAGE_RSYSLOG is not set
+# BR2_KAGE_RTORRENT is not set
+# BR2_KAGE_RUBY is not set
+# BR2_KAGE_RYGEL is not set
+# BR2_KAGE_SAMBA4 is not set
+# BR2_KAGE_SCONESERVER is not set
+# BR2_KAGE_SED is not set
+# BR2_KAGE_SELINUX_PYTHON is not set
+# BR2_KAGE_SETOOLS is not set
+# BR2_KAGE_SHAIRPORT_SYNC is not set
+# BR2_KAGE_SHAPELIB is not set
+# BR2_KAGE_SHARED_MIME_INFO is not set
+# BR2_KAGE_SIGROK_CLI is not set
+# BR2_KAGE_SISPMCTL is not set
+# BR2_KAGE_SMARTMONTOOLS is not set
+# BR2_KAGE_SMSTOOLS3 is not set
+# BR2_KAGE_SNAPPY is not set
+# BR2_KAGE_SNMPPP is not set
+# BR2_KAGE_SNORT is not set
+# BR2_KAGE_SOFTETHER is not set
+# BR2_KAGE_SPDLOG is not set
+# BR2_KAGE_SQUEEZELITE is not set
+# BR2_KAGE_SQUID is not set
+# BR2_KAGE_SSHFS is not set
+# BR2_KAGE_SSLH is not set
+# BR2_KAGE_START_STOP_DAEMON is not set
+# BR2_KAGE_STELLA is not set
+# BR2_KAGE_SURICATA is not set
+# BR2_KAGE_SYSKLOGD is not set
+# BR2_KAGE_SYSLOG_NG is not set
+# BR2_KAGE_TAGLIB is not set
+# BR2_KAGE_TAR is not set
+# BR2_KAGE_TASKD is not set
+# BR2_KAGE_TCLAP is not set
+# BR2_KAGE_TESSERACT_OCR is not set
+# BR2_KAGE_TFTPD is not set
+# BR2_KAGE_THRIFT is not set
+# BR2_KAGE_TIME is not set
+# BR2_KAGE_TINYXML is not set
+# BR2_KAGE_TINYXML2 is not set
+# BR2_KAGE_TMUX is not set
+# BR2_KAGE_TPM2_ABRMD is not set
+# BR2_KAGE_TREE is not set
+# BR2_KAGE_UDFTOOLS is not set
+# BR2_KAGE_UFTP is not set
+# BR2_KAGE_UNRAR is not set
+# BR2_KAGE_UNSCD is not set
+# BR2_KAGE_UNZIP is not set
+# BR2_KAGE_UPMPDCLI is not set
+# BR2_KAGE_URG is not set
+# BR2_KAGE_USSP_PUSH is not set
+# BR2_KAGE_UTIL_LINUX_AGETTY is not set
+# BR2_KAGE_UTIL_LINUX_BFS is not set
+# BR2_KAGE_UTIL_LINUX_BINARIES is not set
+# BR2_KAGE_UTIL_LINUX_CAL is not set
+
```

Comment:
Is it worth disabling some unneeded packages to reduce the size of the default rootfs?

```
BR2_PACKAGE_APACHE
BR2_PACKAGE_APR
BR2_PACKAGE_CPPZMQ
BR2_PACKAGE_EXPAT
BR2_PACKAGE_PHP
BR2_PACKAGE_ZEROMQ
BR2_PACKAGE_ZMQPP
```

Do any FireSim workloads depend on these?

Reply:
I'm not aware of any, I think minimizing the image is definitely worth it if we can get away with it. None of my unit tests use those, but I can't be sure about anyone else.

Reply:
ISCA memcached workload depends on several of those. let's deal with this sort of thing later

Reply:
I'll leave it then